docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/index.md

```diff
@@ -402,6 +402,12 @@ In this section, we'll introduce the following concepts:
         <p>Learn to receive WSS Access logs from the Symantec WSS API.</p>
         </div>
       </div>
+      <div className="box smallbox card">
+        <div className="container">
+        <a href="/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/taxii"><img src={useBaseUrl('img/send-data/taxii_logo.png')} alt="icon" width="60"/><h4>TAXII</h4></a>
+        <p>Learn to collect indicators from threat intel services using the TAXII/STIX 1.1 specification.</p>
```

Comment:
If you run a search for this topic, looks like the preferred nomenclature is STIX/TAXII 

```suggestion
        <p>Learn to collect indicators from threat intel services using the STIX/TAXII 1.1 specification.</p>
```